frontend/package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
+    "start-dev": "ng serve --proxy-config proxy.config.json",
```

Comment:
(so wait, how did you two have both backend and frontend running simultaneously without this)